src/ext/java/org/opentripplanner/ext/legacygraphqlapi/datafetchers/LegacyGraphQLAlertImpl.java

```diff
@@ -114,10 +124,14 @@
         environment.getLocale());
   }
 
-  //TODO
   @Override
   public DataFetcher<Iterable<Map.Entry<String, String>>> alertDescriptionTextTranslations() {
-    return environment -> null;
+    return environment -> {
+      var text = getSource(environment).alertDescriptionText;
+      return text instanceof TranslatedString
```

Comment:
This is duplicated code. Can you use a reusable method?

Reply:
Indeed. I'll do that.

Reply:
I changed that now in https://github.com/opentripplanner/OpenTripPlanner/pull/3747/commits/973c9f370acfd5673e99862bf51592986112dba0